packages/core-utils/src/itinerary.js

```diff
@@ -118,7 +118,7 @@ export function getMapColor(mode) {
   mode = mode || this.get("mode");
   if (mode === "WALK") return "#444";
   if (mode === "BICYCLE") return "#0073e5";
-  if (mode === "SUBWAY") return "#f00";
+  if (mode === "SUBWAY") return "#E60000"; /* from #f00 to copmly with WCAG */
```

Comment:
Lowercase `e`

Reply:
Typo "comply"

Reply:
Fixed in 59cfcfb631e5a0f6c6ef52bc7260faf988bcd5bd